Samples/office-contextual-tabs/src/taskpane/taskpane.html

```diff
@@ -28,6 +28,9 @@
 </head>
 
 <body class="ms-font-m ms-welcome ms-Fabric">
+    <!-- The following image URL tracks diagnostic data for this sample add-in. Please remove the image tag if you reuse this sample in your own code project. -->
+    <img src="https://pnptelemetry.azurewebsites.net/pnp-officeaddins/samples/Office.Contextual-tabs-run" />
```

Comment:
This seems like a different format from the rest.
```suggestion
    <img src="https://pnptelemetry.azurewebsites.net/pnp-officeaddins/samples/office-contextual-tabs-run" />
```